atom/browser/native_window_mac.mm

```diff
@@ -277,7 +278,29 @@ - (NSRect)window:(NSWindow*)window
 
 @end
 
-@interface AtomNSWindow : EventDispatchingWindow {
+@interface AtomPreviewItem : NSObject <QLPreviewItem>
+
+@property (nonatomic, retain) NSURL *previewItemURL;
+@property (nonatomic, retain) NSString *previewItemTitle;
+
+- (id)initWithURL:(NSURL *)anURL title:(NSString *)aTitle;
+
+@end
+
+@implementation AtomPreviewItem
+
+- (id)initWithURL:(NSURL *)url title:(NSString *)title {
```

Comment:
> For vendor/brightray I feel like this occurred when I merged master in my branch. so looks like this was done in a previous commit.

Yeah, it should be reverted though since this pull request doesn't need to update brightray and so it shouldn't show up in the diff at all.